packages/api/cms-api/package.json

```diff
@@ -127,6 +128,7 @@
         "express": "^5.0.0",
         "graphql": "^16.6.0",
         "nest-commander": "^3.0.0",
+        "nodemailer": "^7.0.0|^6.0.0",
```

Comment:
Did you check if we can make that an optional dependency somehow? 

Adding a peer dependency forces all projects to install nodemailer (and counts as a breaking change)

Reply:
@Ben-Ho I tested this locally by starting demo-api without nodemailer installed. This failed for me because it's still imported statically via import:

<img width="829" alt="Bildschirmfoto 2025-06-16 um 09 10 17" src="https://github.com/user-attachments/assets/703bb0c1-50b4-4eb6-a82a-65fb10aa5436" />

I built a dynamic loader function using require in https://github.com/vivid-planet/comet/pull/3969/files. This worked for me

Reply:
We discussed this in last week's meeting and decided to make nodemailer a normal dependency. Please adapt accordingly.